src/pages/errors/no-service-worker.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * Page to display a user friendly message when `navigator.serviceWorker` is not available.
+ */
+
+import React from 'react'
+
+export default function NoServiceWorkerErrorPage (): React.JSX.Element {
+  return (
+    <div>
+      <h1>Service Worker Error</h1>
```

Comment:
should add the header component here and adjust wording. open to thoughts @lidel & @2color